music21/harmony.py

```diff
@@ -1541,22 +1556,21 @@ def _adjustOctaves(self, pitches):
         thirteenths = ['dominant-13th', 'major-13th', 'minor-13th']
 
         if self.chordKind in ninths:
-            pitches[1] = pitch.Pitch(pitches[1].name + str(pitches[1].octave + 1))
+            pitches[1].octave += 1
         elif self.chordKind in elevenths:
-            pitches[1] = pitch.Pitch(pitches[1].name + str(pitches[1].octave + 1))
-            pitches[3] = pitch.Pitch(pitches[3].name + str(pitches[3].octave + 1))
+            pitches[1].octave +=1
+            pitches[3].octave +=1
 
         elif self.chordKind in thirteenths:
-            pitches[1] = pitch.Pitch(pitches[1].name + str(pitches[1].octave + 1))
-            pitches[3] = pitch.Pitch(pitches[3].name + str(pitches[3].octave + 1))
-            pitches[5] = pitch.Pitch(pitches[5].name + str(pitches[5].octave + 1))
+            pitches[1].octave += 1
+            pitches[3].octave += 1
+            pitches[5].octave += 1
         else:
             return pitches
 
-        c = chord.Chord(pitches)
```

Comment:
please revert this -- I prefer one way of doing the sort -- the speed overhead of creating an extra chord is not enough to justify extra documentation and support concerns.

Reply:
What I mean is please remove the separate function `sortDiatonicAscending` outside of the Chord object -- which doesn't return anything and only works on lists not on tuples, etc. I much prefer the existing way of saying "hey chord, sort your pitches diatonically ascending" than having it as a separate function within the `chord` module